src/wsiToDcm.cpp

```diff
@@ -367,7 +376,12 @@ int WsiToDcm::dicomizeTiff(
       adapative_stop_downsampling = true;
     }
     pool.join();
-    generated_dicom_files.clear();
+    if (preferProgressiveDownsampling) {
+      higher_magnifcation_dicom_files.set_dicom_files(
+                                             std::move(generated_dicom_files));
```

Comment:
Does generated_dicom_files also need to be cleared here to prepare for the next iteration or does the std::move take care of that?

Reply:
The move takes care of it.  Move does a efficient move of the memory.  The effect is generated_dicom_files have no files in them after the move and they have been moved to the into higher magnification.

random search, example: 
https://stackoverflow.com/questions/12613428/stl-vector-moving-all-elements-of-a-vector

Reply:
Thanks.